lib/node_modules/@stdlib/math/base/special/exp/package.json

```diff
@@ -66,5 +66,64 @@
     "euler",
     "power",
     "number"
-  ]
+  ],
+  "__stdlib__": {
```

Comment:
@gunjjoshi Would you mind moving this object to

```
"__stdlib__": {
   "scaffold": {...}
}
```

This will help avoid field conflicts with existing usage of `__stdlib__` configuration objects.

Reply:
Done!